data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/AbstractDataBackedRestAPIIntegrationTest.java

```diff
@@ -340,6 +340,17 @@ protected void assertBodyEquals(final Response response, final String body) thro
     assertThat(response.body().string()).isEqualTo(body);
   }
 
+  protected Response getGzipResponse(final String path, final String contentType)
```

Comment:
I think we could stick to the DSL convention and have an override of the `getResponse` method that accepts the value for the Accept-Encoding header.